packages/nuka/src/types.ts

```diff
@@ -428,6 +441,11 @@ export interface InternalCarouselProps {
    */
   swiping: boolean;
 
+  /**
+   * Whether tab pagination is used to set appropriate roles for slides.
+   */
+  tabbed: boolean;
```

Comment:
`tabbed` doesn't exactly align with the "pagination" and "dots" verbiage currently used, but it aligns with WCAG terminology.